INSTALL.txt

```diff
@@ -56,18 +57,35 @@ Running WSO2 API Manager in standalone mode
                 -Dweb.location	The directory into which UI artifacts
                                 included in an Axis2 AAR file are extracted to
 
+
+Secure sensitive information in carbon configuration files
+----------------------------------------------------------
+
+There are sensitive information such as passwords in the carbon configuration.
+You can secure them by using secure vault. Please go through following steps to
+secure them with default mode.
+
+    On Linux: ./ciphertool.sh -Dconfigure
+    On Windows: ./ciphertool.bat -Dconfigure
+
+For more details see
+    https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/logins-and-passwords/working-with-encrypted-passwords/
+
 System Requirements
 =======================
-1. Minimum memory - 2GB
-2. Processor      - Pentium 800MHz or equivalent at minimum
+1. Minimum memory - 4 GB RAM (2 GB for JVM and 2 GB for the operating system)
+2. Processor      - 3 GHz Dual-core Xeon/Opteron (or latest)
 3. Java 1.8 or higher
 4. The Management Console requires you to enable Javascript of the Web browser,
    with MS IE 7. In addition to JavaScript, ActiveX should also be enabled
    with IE. This can be achieved by setting your security level to
    medium or lower.
-5. To compile and run the sample clients, an Ant version is required. Ant 1.7.0
+5. To compile and run the sample clients in <APIM_HOME>/samples, an Ant version is required. Ant 1.7.0
    version is recommended
-6. To build WSO2 AM from the Source distribution, it is necessary that you have
+6. To build WSO2 APIM from the Source distribution, it is necessary that you have
    JDK 1.8 version and Maven 3.0.4 or later
 
+For more details see
+   https://apim.docs.wso2.com/en/next/install-and-setup/install/installation-prerequisites/
```

Comment:
Isn't it ideal to link to the specific doc-version path itself? i.e.
https://apim.docs.wso2.com/en/3.2.0/install-and-setup/install/installation-prerequisites/

Reply:
Fixed https://github.com/wso2/product-apim/pull/8971/commits/061ead8cf43422243492a81c383c1681db180714